src/python/pants/backend/docker/docker_build.py

```diff
@@ -39,11 +46,13 @@ def dockerfile_path(self) -> str:
     def image_name(self) -> str:
         return ":".join(s for s in [self.address.target_name, self.image_version.value] if s)
 
-    def image_tag(self, registries: DockerRegistries) -> str:
-        registry = registries.get(self.registry.value)
-        if registry:
-            return "/".join([registry.address, self.image_name])
-        return self.image_name
+    def image_tags(self, registries: DockerRegistries) -> tuple[str, ...]:
+        image_name = self.image_name
+        registries_options = tuple(registries.get(*(self.registries.value or [])))
+        if not registries_options:
+            return (image_name,)
+
+        return tuple("/".join([registry.address, image_name]) for registry in registries_options)
```

Comment:
So right now every image gets a tag in every globally-configured registry? 

Would it make sense (in a followup change) to let people override this with a `tags=` field on the `docker_image` target? So if you have explicit tags on the target, we don't generate tags in this way? That would also solve https://pantsbuild.slack.com/archives/C046T6T9U/p1632980124236000

Reply:
Every image gets a tag in every globally configured registry, that is tagged with `default: true`, unless there is a `registries=[...]` on the `docker_image` target, in which case, only to those registries explicitly specified.

Reply:
Perfect